Fix a stupid typo preventing to correctly pause non periodic tasks

diff --git server/exec-task.cc server/exec-task.cc
index e6e5979..df69cd6 100644
--- server/exec-task.cc
+++ server/exec-task.cc
@@ -65,9 +65,7 @@ void *
 <'if {[catch {$task period}]} {'>
     if (!(sched & GENOM_SCHED_CONT)) {
       /* timeout after 1s to update rusage */
-      static const struct timespec tout = {
-        tvs.tv_sec + 1,  tvs.tv_usec * 1000
-      };
+      struct timespec tout = { tvs.tv_sec + 1,  tvs.tv_usec * 1000 };
       while (!task.runnable)
         if (pthread_cond_timedwait(&task.sync, &task.lock, &tout))
           break;
